src/lib_json/json_writer.cpp

```diff
@@ -1157,6 +1172,14 @@ StreamWriter* StreamWriterBuilder::newStreamWriter() const
   } else {
     throwRuntimeError("commentStyle must be 'All' or 'None'");
   }
+  PrecisionType precisionType(significantDigits);
+  if (pt_str == "significant") {
```

Comment:
These names seem long and ambiguous. Maybe `exp` and `noexp` are better names. Or something else?

Reply:
I used `significant` word for old-style precision formatting. see https://github.com/open-source-parsers/jsoncpp/pull/752#discussion_r173970681. Answer it please. If we delete old-style formatting - then of course it is better change names.